DataConnectors/Symantec DLP/Connector_Syslog_SymantecDLP.json

```diff
@@ -0,0 +1,94 @@
+{
+    "id": "SymantecDLP",
+    "title": "Symantec DLP",
+    "publisher": "Broadcom",
+    "descriptionMarkdown": "The [Broadcom Symantec Data Loss Prevention (DLP)](https://www.broadcom.com/products/cyber-security/information-protection/data-loss-prevention) connector allows you to easily connect your Symantec DLP with Azure Sentinel, to create custom dashboards, alerts, and improve investigation. This gives you more insight into your organization’s information, where it travels, and improves your security operation capabilities.",
+    "additionalRequirementBanner":"These queries are dependent on a parser based on a Kusto Function to work as expected. Follow the steps to use this Kusto functions alias **SymantecDLP** in queries and workbooks. [Follow these steps to get this Kusto functions.](https://aka.ms/sentinel-symantecdlp-parser)",
+    "graphQueries": [{
+        "metricName": "Total data received",
+        "legend": "Symantec DLP",
```

Comment:
Legend value should be SymantecDLP (remove space) as it is DATATYPE_NAME. Refer https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Templates/Connector_CEF_template.json.

Reply:
changes have been pushed